docs/benchmarks/test-results-summary.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Performance Benchmarks and Test Reports
+---
+
+
+## Introduction
+This page summarizes key performance metrics for nwaku and provides links to detailed test reports.
```

Comment:
```suggestion
This page summarises key performance metrics for nwaku and provides links to detailed test reports.
```

Slightly controversial, but we've been asked to switch to British English when we publish. We certainly don't follow this consistently elsewhere, so up to you.